common/assets/scss/overrides/_visibility.scss

```diff
@@ -0,0 +1,3 @@
+.is-hidden {
```

Comment:
There's a class in the Design System that will already do this for us: `.govuk-!-display-none`

Reply:
As discussed I have added a `TODO` around this and changed the name to `govuk-!-display-none`. Currently this work has been done but has not yet been released. I am watching hte releases and will get this sorted once its out in the wild.

[commit info GOVUK-frontend](https://github.com/alphagov/govuk-frontend/commit/6bbee4b116525199bc2e07659b9af90722555ba2)